airflow/cli/cli_config.py

```diff
@@ -1721,6 +1747,12 @@ class GroupCommand(NamedTuple):
         func=lazy_load_command("airflow.cli.commands.db_command.drop_archived"),
         args=(ARG_DB_TABLES, ARG_YES),
     ),
+    ActionCommand(
```

Comment:
Should we already move it to the connections command?

Reply:
Yes, on it!

Reply:
I hope it is fine now to replace every instance of `db init` and `db upgrade` with `db migrate` in the doc?

Reply:
There are few places where `initdb` from `airflow.utils.db` is being called that creates default connection ex - 
https://github.com/apache/airflow/blob/f91c003092e0e2c0e5b9a5073352b023a51a887f/airflow/utils/db.py#L1669-L1670

https://github.com/apache/airflow/blob/f91c003092e0e2c0e5b9a5073352b023a51a887f/airflow/utils/db.py#L845-L848

Shall we replace these now with `upgradedb` or maybe in later commits when we would have added the desired functionality to `create_default_connections`?

Reply:
> I hope it is fine now to replace every instance of db init and db upgrade with db migrate in the doc?

yes

> Shall we replace these now with upgradedb or maybe in later commits when we would have added the desired functionality to create_default_connections?

should be `migrate`  in both cases I think. That was the intention in both cases, I think

Reply:
> should be `migrate` in both cases I think.

Currently, we have just renamed the command and the function names in `db_commands.py`. I think its best to take the renaming of `airflow.utils.db.upgradedb` to be taken in another PR since it is used at lots of other places and quite some refactoring is required for above change.